lib/config/typescript.js

```diff
@@ -35,15 +33,33 @@ module.exports = {
         'no-useless-constructor': 'off',
         'no-shadow': 'off',
         'no-use-before-define': 'off',
+        semi: 'off',
```

Comment:
Same comment here, I feel overrides are making it difficult for people to later customize the config